raster/r.kappa/prt_kappa.c

```diff
@@ -0,0 +1,57 @@
+#include <stdlib.h>
+#include <grass/gis.h>
+#include <grass/glocale.h>
+#include "kappa.h"
+#include "local_proto.h"
+
+
+void prt_kappa(void)
```

Comment:
While _strč prst skrz krk_ is much more popular Czech and Slovak tongue-twister, this reminded me of _krt prd skrz drn, zprv zhlt hrst zrn_. In other words, I would prefer `print_kappa` here.

Reply:
All functions and files are renamed.